lld/test/wasm/Inputs/allow-multiple-definition.s

```diff
@@ -0,0 +1,6 @@
+  .hidden foo
```

Comment:
FYI In ELF, we now migrate used-once extra files from Inputs/ to `split-file`

Reply:
Should that be included in this PR? It seems to me that it's better to leave it same as ELF test for easier tracking.

Reply:
I think matching ELF for now seem fine.  We can try to follow the migration there as it happens.